### PR TITLE
Add webhook payload metric

### PR DESCRIPTION
**Pull request by tarakpurohit**

### CODE REVIEW

File: src/github/middleware.ts [hunk 1]
```diff
@@ -77,6 +77,10 @@ export default (
 		enhanceOctokit(context.github);
 		const webhookEvent = extractWebhookEventNameFromContext(context);
 
+		// Metrics for payload size
+		emitWebhookPayloadMetrics(webhookEvent,
+			Buffer.byteLength(JSON.stringify(context.payload), "utf-8"));
```
Comment (KAbakumovAtl) [inline src/github/middleware.ts:82]: Can we emit this metric before serializing the palyoad again?

We should either do it before the request was serialised or use req. parameters:
`req.get("content-length")` or `req.socket.bytesRead` if content-length is missing

File: src/util/webhooks.ts [hunk 2]
```diff
@@ -87,3 +87,15 @@ export const emitWebhookFailedMetrics = (webhookName: string) => {
 	statsd.increment(metricWebhooks.webhookFailure, tags);
 
 }
+
+export const emitWebhookPayloadMetrics = (webhookName: string, size: number) => {
+	const tags = {
+		name: webhookName,
+		payloadSize: size.toString()
+	};
+	statsd.histogram(
```
Comment (KAbakumovAtl) [inline src/util/webhooks.ts:96]: So, hence the purpose is to identify how many webhooks are there with a pyload > 128KB 

we should use a bucketed histogram metric here (See how it is done for the latencies metrics, we hse gsd_histogram parameter where we specify buckets sizes)

File: src/util/webhooks.ts [hunk 3]
```diff
@@ -87,3 +87,15 @@ export const emitWebhookFailedMetrics = (webhookName: string) => {
 	statsd.increment(metricWebhooks.webhookFailure, tags);
 
 }
+
+export const emitWebhookPayloadMetrics = (webhookName: string, size: number) => {
+	const tags = {
+		name: webhookName,
+		payloadSize: size.toString()
```
Comment (KAbakumovAtl) [inline src/util/webhooks.ts:94]: We must never use values with unbounded variations as tags. It will explode the metric cardinality

**Review by KAbakumovAtl — CHANGES_REQUESTED** [review]
I have to request changes because of the metric cardinality. 

That's what blocks the PR:
- [x] Do not use payloadSize as a tag

File: src/github/middleware.ts (hunk 1 above)
Comment (tarakpurohit) [reply src/github/middleware.ts:82]: We dont have access to `req` for webhook events, hence, we have to use the payload from context

**Review by KAbakumovAtl — APPROVED** [review]

File: src/util/webhooks.ts (hunk 2 above)
Comment (tarakpurohit) [reply src/util/webhooks.ts:101]: Added bucket sizes.

**Review by atrigueiro — APPROVED** [review]

**Review by thombergs — APPROVED** [review]

**Pull request merged**